src/components/ArticleForm.js

```diff
@@ -1,56 +1,65 @@
-import React from "react"
-import agent from "../agent"
-import { connect } from "react-redux"
+import React from "react";
+import agent from "../agent";
+import { connect } from "react-redux";
 
 // create foprm page for creating posts, editing and deleting posts
 
-const mapStateToProps = state => ({}) 
+const mapStateToProps = state => ({});
 const mapDispatchToProps = dispatch => ({
+  onChangeTitle: value =>
+    dispatch({ type: "UPDATE_ARTICLE_FIELDS", key: "title", value }),
```

Comment:
You should consider having those `type` in a constant exported from a file in your redux folder. I'm talking about all the action types such as `UPDATE_ARTICLE_FIELDS`

And I'm assuming `value` is the `payload`

Reply:
ahh like _doc_strings_ you mean?

yes the value is payload here.